index.js

```diff
@@ -211,30 +225,22 @@ internals.log = function(namespace, level, contextId, message, data) {
     }
 
     contextId = contextId || module.exports.id()
-    const time = new Date().toISOString()
-    const output = Object.assign({}, internals.globalContext, { level, time, namespace, contextId })
+    const time = new Date()
+    const output = {}
+    output.meta = Object.assign({}, internals.globalContext, { level, time, namespace, contextId })
     if (message) output.message = message
     if (data) output.data = data
     internals.write(output)
 }
 
 /**
- * Write output
- * @param {Object} output
+ * Write output using all enabled outputs
+ * @param {LogOutput} output
  */
 internals.write = function(output) {
-    let result = ''
-    if (internals.output === 'pretty') {
-        const prefix = `${output.message}\n`
-        result = `${prefix}${prettyOutput(output, { maxDepth: 6 }, 2)}`
-    } else if (internals.output === 'json') {
-        const backup = Error.prototype.toJSON
-        Error.prototype.toJSON = internals.errorToJson
-        result = JSON.stringify(output)
-        Error.prototype.toJSON = backup
-    }
-    process.stdout.write(result)
-    process.stdout.write('\n')
+    _.forEach(internals.outputs, outputFn => {
```

Comment:
Why not `internals.outputs.forEach`?

Reply:
Both lodash functions and native functions are being used, I think it's best to prefer native ones over lodash, it's easier for people to contribute without having to know about tens of lodash helpers while the features are already available, also regarding  performance, v8 is continuously improving on that matter, taking in account latest js features, for example `{ ...thingToCopy }` VS `_.assign({}, thingToCopy)`, IMHO the less we rely on third party modules, the best it is.

Reply:
By the way on benchmarks, Object.assign is the quickest of all. Updated. Done